project/build.scala

```diff
@@ -125,12 +136,12 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm, ntriples_jvm)
```

Comment:
@InTheNow thanks for investigating.

@antonkulaga can you please remove the unnecessary dependencies as part of this PR?